cmd/knit/rest/client.go

```diff
@@ -204,28 +215,32 @@ type KnitClient interface {
 	// - error
 	GetRunLog(ctx context.Context, runId string, follow bool) (io.ReadCloser, error)
 
-	// FindRun find run with given planId, knitId and status.
+	// FindRun find run with RunFindQuery.
```

Comment:
I suppose

```suggestion
	// FindRun find run with RunFindParameter.
```

Or, change typename from `FindRunParameter` to `FindRunQuery`.